src/app/children/child-details/child-details.component.html

```diff
@@ -297,7 +297,7 @@ <h3>Coaching Attendance</h3>
         </mat-form-field>
       </div>
 
-      <div>
+      <!-- <div>
```

Comment:
Don't forget to delete the commented code before the final merge

Reply:
I cleaned up all the unnecessary comments now